wagtail/wagtailcore/blocks/list_block.py

```diff
@@ -7,18 +7,29 @@
 from django.template.loader import render_to_string
 from django.utils.html import format_html, format_html_join
 from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext as _
 
 from wagtail.wagtailcore.utils import escape_script
 
 from .base import Block
 from .utils import js_dict
 
-__all__ = ['ListBlock']
+__all__ = ['ListBlock', 'ListBlockValidationError']
+
+
+class ListBlockValidationError(ValidationError):
+    def __init__(self, block_errors=None, non_block_errors=None):
+        self.block_errors = block_errors
+        self.non_block_errors = non_block_errors
+        super(ListBlockValidationError, self).__init__('Validation error in ListBlock')
 
 
 class ListBlock(Block):
 
-    def __init__(self, child_block, **kwargs):
+    def __init__(self, child_block, max_num=None, min_num=None, **kwargs):
```

Comment:
Just a minor suggestion.

We can rename these to `min_length` and `max_length` to be consistent with what various fields in Django uses.

Reply:
@CuriousLearner That'll make them inconsistent with Wagtail though... http://docs.wagtail.io/en/v2.7/topics/streamfield.html#streamblock

Reply:
Ah, I see. Fair enough.

Thank you for pointing me to that :)